### ISSUE TITLE
Feature request: Support nested groups on the Repositories page

### ISSUE THREAD

**Issue by gitblit** [gitblit/gitblit#725]
Originally reported on Google Code with ID 429

```
In my organization we have a lot of Git repositories and would like to be able to group
the repositories in a nested fashion, e.g.

Managers
- Manager X
  - Server
  - Client
- Manager Y
  - Server
  - Database
  - Client
etc.

Here https://code.google.com/p/gitblit/issues/detail?id=406#c1 it is mentioned that
it's possible to nest repos with '/', so what I'm suggesting is really just that the
interface reflects this tree structure more clearly than now. Furthermore the GUI change
here https://code.google.com/p/gitblit/issues/detail?id=231&q=collapse#makechanges,
will fit nicely into this feature.


```

Reported by `jakob@jboysen.dk` on 2014-05-20 11:04:35

**Comment by fzs** [comment]
Google Code issue 406 was imported as issue #702.
Google Code issue 231 was imported as issue #527.

**Comment by fzs** [comment]
This is implemented by PR #1267
